src/mixins/filing-template-mixin.ts

```diff
@@ -76,7 +76,7 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
   @Action(useStore) setBusinessContact!: (x: ContactPointIF) => void
   @Action(useStore) setCertifyState!: (x: CertifyIF) => void
   @Action(useStore) setCooperativeType!: (x: CoopTypes) => void
-  @Action(useStore) setCorrectNameOption!: (x: CorrectNameOptions) => void
+  // @Action(useStore) setCorrectNameOption!: (x: CorrectNameOptions) => void
```

Comment:
VS Code was giving me a warning that this action is already imported (via another mixin). Instead of deleting the declaration here, I just commented it out. This should help a curious developer understand why this declaration is not needed.